docs/processors/getting-started.mdx

```diff
@@ -4,99 +4,156 @@ sidebar_label: "Getting Started"
+
+## How to use a processor for your pipeline or connector
+
+In these following examples, we're using the [`json.decode`](/docs/processors/builtin/json.decode), but you could use any other you'd like from our [Built-in](/docs/processors/builtin/) ones, or even [reference](/docs/processors/referencing) your own [Standalone processor]((/docs/processors/standalone-processors)).
 
-Given that every processor can have one (and only one) parent, processors cannot be shared. In case the same processing
-needs to happen for different sources or destinations, you have two options:
+:::info
 
-1. If records from all sources (or all destinations) need to be processed in the same way, then you can create
-   a pipeline processor
-2. If records from some, but not all, sources (or destinations) need to be processed in the same way, then you need to
-   create multiple processors (one for each source or destination) and configure them in the same way.
+When referencing the name of a processor plugin there are different ways you can make sure you're using the one you'd like. Please, check out the [Referencing Processors](/docs/processors/referencing) documentation for more information.
 
-## Creating a processor through a pipeline configuration file
+:::
+
+
+### Through a [pipeline configuration file](/docs/pipeline-configuration-files/getting-started)
+
+#### Using a pipeline processor
 
 Creating a pipeline processor through a pipeline configuration file can be done as below:
 
 ```yaml
-version: 2.0
+version: 2.2
 pipelines:
   - id: example-pipeline
     connectors:
     # define source and destination connectors
     # ...
     processors:
       - id: extract-name
-        type: extractfieldpayload
+        plugin: json.decode
         settings:
           field: name
 ```
 
+#### Using a connector processor
+
 Similarly, we can configure a connector processor, i.e. a processor attached to a connector:
 
 ```yaml
-version: 2.0
+version: 2.2
 pipelines:
   - id: example-pipeline
     connectors:
       - id: conn1
         # other connector configuration
         processors:
           - id: extract-name
-            type: extractfieldpayload
+            plugin: json.decode
             settings:
               field: name
       # other connectors
 ```
 
 The documentation about pipeline configuration files can be found [here](/docs/configuration/pipeline-configuration-files).
 
-## Creating a processor through the HTTP API
+### Through the [HTTP API](/api)
+
+The processor endpoints live under the `/v1/processors` namespace, and if you'd like to use our [HTTP API](/api) to attach a processor to either connector or a pipeline, here's how you could do it.
+
+#### Using a pipeline processor
+
+```json lines
+POST /v1/processor
+{
+  "parent": {
+    "type": "TYPE_PIPELINE",
+    "id": "pipeline-id"
+  },
+  "config": {
+    "settings": {
+      "field": "name"
+    },
+    "workers": 0
+  },
+  "condition": "string",
+  "plugin": "json.decode"
+}
+```
```

Comment:
I propose getting rid of one of these examples, if not even both 😱 The thing is that using the API is kinda the _old_ way of doing things, we don't need to promote it more than we should. We can mention it and link to the API docs, but people should really use the pipeline configuration file.

Reply:
Got it. I'll re-structure these.